traefik/templates/rbac/clusterrole.yaml

```diff
@@ -1,4 +1,9 @@
+{{- if .Values.rbac.enabled -}}
+{{- if .Values.rbac.namespaced }}
```

Comment:
Could you please split this into 2 file. 
- clusterrole.yaml
- role.yaml

Reply:
Done